onboard/frontend/ui/Menu.cs

```diff
@@ -216,11 +181,11 @@ public class Menu : IMenu {
                 // don't download the banner for the default game
                 Client.downloadBanner(game.id);
             } // check if /tmp/ has the banner
-            
-            string bannerPath = $"{Env.get("DEVCADE_PATH").unwrap_or_else(() => Env.get("HOME").unwrap() + "/.devcade")}/{game.id}/banner.png";
+
+            string bannerPath = $"/tmp/devcade/{game.id}/banner.png";
```

Comment:
This might want to not be reverted. It will probably work with it reverted tho because i symlinked tmp/devcade to ~.devcade/.